src/components/content-tab-assistant.tsx

```diff
@@ -158,11 +159,19 @@ export const Message = ( { children, isUser, className }: MessageProps ) => {
 			) }
 		>
 			<div
+				id={ id }
+				role="group"
+				aria-labelledby={ id }
 				className={ cx(
 					'inline-block p-3 rounded border border-gray-300 lg:max-w-[70%] select-text',
 					! isUser ? 'bg-white' : 'bg-white/45'
 				) }
 			>
+				<div className="relative">
+					<span className="sr-only">
```

Comment:
Wrapping an element leveraging Tailwind's `sr-only` class is [necessary to avoid](https://github.com/tailwindlabs/tailwindcss/discussions/12429#discussioncomment-7587178) unexpected overflow scrolling for the top level document.